public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -727,7 +727,7 @@ export const Explorer = ({
       if (availability !== true) {
         await updateQueryInStore(tempQuery);
       }
-      await fetchData();
+      await fetchData(startTime, endTime);
```

Comment:
Can you create an issue/discussion point regarding this change? I think with this change all instances of fetchdata will be called with startTime/endTime, so we can remove the dependency on startTime/endTime from the query in redux - or vice versa. In my opinion it is bad to have two references/sources of truth to anything (timerange/patterns, etc.), so either we should remove it from the redux query if it is not being used, or only rely on time from query in redux - not both @mengweieric thoughts?

Reply:
Agreed with an issue to track and document this and to have a consistent way of set time range in both app and explorer sides. We may want to remove explicit start/end time dependency for explorer and instead, dispatch timerange to app query state. This only does the trick to make it work but would potentially cause more issues and bugs